Dev tosuwu

BadiiiiX opened the pull request.

tosu-dev committed:
Starting

Topbar and navbar components created with all elements needed.
Home page started and error 404 page started

tosu-dev committed:
Update PageNotFound.vue

tosu-dev committed:
Merge pull request #2 from tosu-dev/dev-tosuwu

Dev tosuwu

tosu-dev committed:
Add Hero

Add hero with bg, shadow, bottom cut, title, search bar, search button.
Remove topbar and navbar in page not found.

tosu-dev committed:
Merge pull request #3 from tosu-dev/dev-tosuwu

Add Hero

tosu-dev committed:
Add hover effect to nav items, and style error 404 page

tosu-dev committed:
Merge pull request #4 from tosu-dev/dev-tosuwu

nav items hover

BadiiiiX commented on src/App.vue:
```diff
@@ -1,91 +1,16 @@
-  </header>
+  <div v-if="this.$route.name != 'PageNotFound'">
+    <TopBar></TopBar>
+    <NavBar></NavBar>
```
Faire ici un seul component, beaucoup plus lisible

tosu-dev replied:
J'ai regroupé les 2 components TopBar et Navbar, dans un component TopBar

BadiiiiX commented on src/assets/css/base.css:
```diff
@@ -0,0 +1,13 @@
+/* color palette from <https://github.com/vuejs/theme> */
```
remove that

BadiiiiX commented on src/assets/css/main.css:
```diff
@@ -0,0 +1,5 @@
+@import "base.css";
+
+#app {
```
Useless

BadiiiiX commented on src/components/Hero.vue:
```diff
@@ -0,0 +1,71 @@
+  height: 0;
+  width: 0;
+  border-bottom: 200px solid var(--white);
+  border-left: 1920px solid transparent;
```
Revoir ça, peut overflow la page

tosu-dev replied:
Solution, faire un triangle en svg

BadiiiiX commented on src/components/Hero.vue:
```diff
@@ -0,0 +1,71 @@
+}
+
+.hero-title {
+  width: 80vh;
```
voir le responsive de ça

tosu-dev replied:
le responsive ne marche pas bien quand on reduit l'écran

BadiiiiX commented on src/components/Hero.vue:
```diff
@@ -0,0 +1,71 @@
+  margin: auto;
+  margin-top: 24px;
+  width: 312px;
+  border: 3px solid var(--light-blue);
```
4 ou 2px, toujours des multiples de 2.

BadiiiiX commented on src/components/HeroSearchBar.vue:
```diff
@@ -0,0 +1,47 @@
+<template>
+  <div class="searchbar-container">
+    <input type="text" placeholder="Rechercher..." />
+    <button><IconSearch></IconSearch></button>
```
Mettre le bouton dans le IconSearch **Si possible**

tosu-dev replied:
T'as mal compris 'spece de Hmar

BadiiiiX commented on src/components/HeroSearchBar.vue:
```diff
@@ -0,0 +1,47 @@
+  padding: 0;
+  height: 64px;
+  border: 0;
+  border-radius: 16px 0 0 16px;
```
Utiliser les border-radius-<top/bottom>

tosu-dev replied:
L'attribut css serait -moz-border-raadius-<top-left/bottom-left>, donc nope.

BadiiiiX commented on src/components/NavBar.vue:
```diff
@@ -0,0 +1,45 @@
+    />
+    <nav>
+      <NavBarItem text="L'agglo" href="/"></NavBarItem>
+      <NavBarItem text="Text" href="#"></NavBarItem>
```
Optimisation: Mettre un texte par défaut ?

tosu-dev replied:
J'ai essayé avec un "default", ça ne marche pas

BadiiiiX commented on src/components/NavBarItem.vue:
```diff
@@ -0,0 +1,57 @@
+  bottom: 0;
+  background: var(--light-blue);
+  height: 8px;
+  -webkit-transition-property: right;
```
à enlever

BadiiiiX commented on src/components/NavBarItem.vue:
```diff
@@ -0,0 +1,57 @@
+  /* hvr-underline-from-left */
+  display: inline-block;
+  vertical-align: middle;
+  -webkit-transform: perspective(1px) translateZ(0);
```
à enlever

BadiiiiX reviewed (commented):
Le code est bon, en grande partie.

Retirer les -webkit, et régler ce qui est réglable (lire les 11 commentaires que j'ai mis et les valider)

Je commit après :)

tosu-dev committed:
Corrections

Remove hero-cut
Reset default css
New logo
Start quick access
Add SelectionTitle
Add TopPage = TopBar + NavBar
